### ISSUE TITLE
ZopfliPNG error: Decompress Zlib data failed.

### ISSUE THREAD

**Issue by f2d** [JayXon/Leanify#62]
Hi.

While trying to optimize one ORA file saved years ago in MyPaint (which overall went very well, 7 MB cut down to 4), I got an error message about one PNG inside:
```
-> data/layer200.png
ZopfliPNG failed!
Decompress Zlib data failed.
```
Running Leanify directly on this extracted PNG gives the same message and the file is unchanged, only modtime.
I tried all 4 versions of the last build (243) from https://ci.appveyor.com/project/JayXon/leanify - same error message.

OptiPNG, OxiPNG, TruePNG all have no problem optimizing it.
Running Leanify directly after them also has no problems and saves some more bytes (same result as OxiPNG -Z alone, but faster).

I'm attaching the unoptimized PNG file here.
![layer200.png](https://user-images.githubusercontent.com/4068475/99190769-87ee4e80-2779-11eb-9c11-4d71390f7f8b.png)

**Comment by JayXon** [comment]
If you use `-v` it will show that the png has incorrect ADLER32
```
Decoding error 58: invalid ADLER32 encountered (checking ADLER32 can be disabled)
```
Maybe I should disable the ADLER32 check in lodepng, since PNG already has CRC32 check for the IDAT.

**Comment by JayXon** [comment]
Turns out the file is not corrupt at all, and it was a bug in `lodepng` that's already fixed.
I've updated the `lodepng` copy in leanify and opened a PR to update `lodepng` in `zopfli`: https://github.com/google/zopfli/pull/176

**Comment by f2d** [comment]
It works now, and somehow cuts 2 bytes more than oxipng 4.0.0 (with -Z) from this PNG file.

By the way, Windows Defender says nothing about build 243, but reports all 244 artifacts as `Trojan:Win32/Spursint.F!cl` and deletes the downloaded files right away.

VirusTotal results are different with each other, so I guess it must be a false-positive.

In order of scanning:
win_x64_MSVC - [1 of 70](https://www.virustotal.com/gui/file/16f3ddf71af406d94cde81b7244c8456bc473326c0133eda1a7db84a52a556a9/detection) (mistaken version in filename here, but the file is right)
win_x64_Clang - [1 of 71](https://www.virustotal.com/gui/file/3544ed6cd552f95cc849d17dba8beb61e5da08b27e6965be44e4d13ea418179e/detection)
win_x86_Clang - [3 of 71](https://www.virustotal.com/gui/file/7dfa1fd09b6e4cbf6b415cef5434684d8e3b94c9cbc9d7bfbc4ccd3188f566ac/detection)
win_x86_MSVC - [5 of 71](https://www.virustotal.com/gui/file/53294301e53dab95bbc5320481d9e1bac64a4bdc319303a3fee73540e785d471/detection)

I allowed it as an exclusion for now, but not sure what else I can do.

Anecdotally, Windows Defender complaining about Leanify - one single thing out of a mega collection of optimizers - was how I found this project in the first place, and it was exactly what I needed.
